package.json

```diff
@@ -578,10 +598,40 @@
                     "group": "0@2"
                 },
                 {
-                    "command": "aws.copyArn",
-                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsCloudFormationFunctionNode|awsStateMachineNode|awsCloudFormationNode)$/",
+                    "command": "aws.s3.downloadFileAs",
+                    "when": "view == aws.explorer && viewItem == awsS3FileNode",
+                    "group": "0@1"
```

Comment:
I'm not sure what these context menus look like, but similar commands should be grouped together between separators. Something like   (download, upload) | (create bucket, folder) | (copy X)

Reply:
I tried to put the groups in the description. Thanks for reminding me, was going to put create bucket / folder into its own section. Right now I have it grouped with upload.